api/src/main/java/io/minio/PostPolicy.java

```diff
@@ -187,7 +217,7 @@ public String bucketName() {
     formData.put("x-amz-algorithm", ALGORITHM);
 
     DateTime date = new DateTime();
-    String region = BucketRegionCache.INSTANCE.region(this.bucketName);
+    String region = this.region != null ? this.region : BucketRegionCache.INSTANCE.region(this.bucketName);
```

Comment:
No, if PostPolicy is used independently (without the Client), BucketRegionCache is inaccessible